### PR TITLE
Install tools for prepare-commit.sh

### PR DESCRIPTION
**Pull request by Gustry**

### CODE REVIEW

File: development/README.md [hunk 1]
```diff
@@ -616,3 +616,11 @@ PYTHONPATH=$PYTHONPATH:/Users/timlinux/Applications/QGIS.app/contents/Resources/
 ```
 $PYTHONPATH:/Users/timlinux/Applications/QGIS.app/contents/Resources/python:/Users/timlinux//Applications/QGIS.app/Contents/Resources/python/plugins/
 ```
+
+### Commit some changes to QGIS
+
+Before committing your changes to QGIS, you need to run `scripts/prepare-commit.sh`. You need to install some tools first:
+```
+brew install gnu-sed
+pip install autopep8
```
Comment (dakcarto) [inline development/README.md]: Hi @Gustry, thanks for contributing!

This should be `pip3`, yes?

**Comment by Gustry** [comment]
Fixed, thanks.
I added a gitignore too

**Comment by dakcarto** [comment]
@Gustry, thanks for contributing!

**Comment by Gustry** [comment]
Thanks to you for this awesome work. Qgis dev on mac is nearly very good ;-)

**Pull request merged**